Tests/feaLib/builder_test.py

```diff
@@ -154,6 +156,7 @@ def check_fea2fea_file(self, name, base=None, parser=Parser):
                 sys.stderr.write(line+"\n")
             self.fail("Fea2Fea output is different from expected. "
                       "Generated:\n{}\n".format("\n".join(actual)))
+        parser.ignore_comments = temp
```

Comment:
No because then you are introducing a redundant exception capture, which would require re-raising any exceptions raised in the code inside. Just go with a temporary variable.